tensorflow/python/keras/benchmarks/keras_examples_benchmarks/antirectifier_benchmark_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Benchmarks on Antirectifier."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from tensorflow.python.keras.benchmarks import benchmark_util
+
+
+class AntirectifierBenchmark(tf.test.Benchmark):
+  """Benchmarks for Antirectifier using `tf.test.Benchmark`."""
+  # Required Arguments for measure_performance.
+  #   x: Input data, it could be Numpy or load from tfds.
+  #   y: Target data. If `x` is a dataset, generator instance,
+  #      `y` should not be specified.
+  #   loss: Loss function for model.
+  #   optimizer: Optimizer for model.
+  #   Other details can see in `measure_performance()` method of
+  #   benchmark_util.
+
+  def __init__(self):
+    super(AntirectifierBenchmark, self).__init__()
+    (self.x_train, self.y_train), _ = tf.keras.datasets.mnist.load_data()
+    self.x_train = self.x_train.reshape(-1, 784)
+    self.x_train = self.x_train.astype("float32") / 255
+
+  def _build_model(self):
+    """Model from https://keras.io/examples/keras_recipes/antirectifier/."""
+    model = tf.keras.Sequential(
+        [
+            tf.keras.Input(shape=(784,)),
+            tf.keras.layers.Dense(256),
+            Antirectifier(),
+            tf.keras.layers.Dense(256),
+            Antirectifier(),
+            tf.keras.layers.Dropout(0.5),
+            tf.keras.layers.Dense(10),
+        ]
+    )
+    return model
+
```

Comment:
Move the above comments on \`measure_performance` here. Maybe rephrase it as something like:

\# In each benchmark test, the required arguments for the method \`measure_performance\` include:
\# ....
\# ....